### PR TITLE
genpolicy: deny UpdateEphemeralMountsRequest

### PR DESCRIPTION
**Pull request by Redent0r**
* genpolicy: deny UpdateEphemeralMountsRequest

Deny UpdateEphemeralMountsRequest by default, because paths to critical Guest components can be redirected using such request.

Upstreaming from https://github.com/microsoft/kata-containers/pull/126

### CODE REVIEW

File: src/tools/genpolicy/rules.rego [hunk 1]
```diff
@@ -38,8 +38,7 @@ default StartTracingRequest := false
 default StatsContainerRequest := true
 default StopTracingRequest := false
 default TtyWinResizeRequest := true
-default UpdateContainerRequest := false
```
Comment (danmihai1) [inline src/tools/genpolicy/rules.rego:41]: UpdateContainerRequest seems unrelated to this PR - let's keep it here.

**Review by danmihai1 — CHANGES_REQUESTED** [review]

File: src/tools/genpolicy/rules.rego (hunk 1 above)
Comment (Redent0r) [reply src/tools/genpolicy/rules.rego:41]: Good catch! updated

**Comment by danmihai1** [comment]
Let's test these changes on TDX, SEV, SNP too. It's possible that they still need to call UpdateEphemeralMounts.

**Review by danmihai1 — APPROVED** [review]

**Review by GabyCT — APPROVED** [review]
